### PR TITLE
v1.0 ios

### PR DESCRIPTION
**Pull request by laves**

### CODE REVIEW

**Comment by laves** [comment]
markdown link will fail until Cocoapod is released

File: binding/ios/KoalaAppTest/KoalaAppTestUITests/KoalaAppTestUITests.swift [hunk 1]
```diff
@@ -0,0 +1,144 @@
...
+    }
+
+    func testPureNoise() throws {
+        let noisePcm = try getPcm(fileUrl: noiseAudioUrl)
+
+        koala = try Koala(accessKey: accessKey)
+        try runTest(inputPcm: noisePcm)
+    }
+
+    func testPureMixed() throws {
```
Comment (ErisMik) [inline binding/ios/KoalaAppTest/KoalaAppTestUITests/KoalaAppTestUITests.swift:93]: I think `PureMixed` is an oxymoron here :P

**Review by ErisMik — APPROVED** [review]

File: binding/ios/KoalaAppTest/KoalaAppTest/AppDelegate.swift [hunk 2]
```diff
@@ -0,0 +1,19 @@
+//
+//  Copyright 2022 Picovoice Inc.
```
Comment (ksyeo1010) [inline binding/ios/KoalaAppTest/KoalaAppTest/AppDelegate.swift:2]:
```suggestion
//  Copyright 2023 Picovoice Inc.
```

File: binding/ios/KoalaAppTest/KoalaAppTest/ViewController.swift [hunk 3]
```diff
@@ -0,0 +1,20 @@
+//
+//  Copyright 2022 Picovoice Inc.
```
Comment (ksyeo1010) [inline binding/ios/KoalaAppTest/KoalaAppTest/ViewController.swift:2]:
```suggestion
//  Copyright 2023 Picovoice Inc.
```

**Review by ksyeo1010 — APPROVED** [review]

**Comment by laves** [comment]
markdown links won't fail once merged to main

**Pull request merged**